nixos/modules/services/network-filesystems/ceph.nix

```diff
@@ -403,7 +402,11 @@ in
     systemd.tmpfiles.rules = [
       "d /etc/ceph - ceph ceph - -"
       "d /run/ceph 0770 ceph ceph -"
-      "d /var/lib/ceph - ceph ceph - -"
-    ];
+      "d /var/lib/ceph - ceph ceph - -"]
```

Comment:
You're missing a map for the mon daemons, they follow the same pattern as the other daemons.